mmdet/core/bbox/assigners/atss_cost_assigner.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from ..builder import BBOX_ASSIGNERS
+from ..iou_calculators import build_iou_calculator
+from .assign_result import AssignResult
+from .base_assigner import BaseAssigner
+
+
+@BBOX_ASSIGNERS.register_module()
+class ATSSCostAssigner(BaseAssigner):
+    """Assign a corresponding gt bbox or background to each bbox.
+
+    ATSSCostAssigner is similar with ATSSAssigner, except that xxx uses a parameter ATSSCostAssigner
```

Comment:
This sentence looks strange.

Reply:
the last commit was right. Please look the commit [`[Feature] modify ddod code0301 extra`](https://github.com/open-mmlab/mmdetection/pull/7279/commits/6e9a185b4c648ac87c2e335a753a3962c838ec4a).